www-root/content/js/main.js

```diff
@@ -16,8 +16,12 @@ $(document).ready(function() {
 	if(Cookies.get('lang')){
 		$langSelector.val(Cookies.get('lang'));
 	} else if(navigator.languages.length){
+		const fallbackLanguage = 'en';
 		var prefLanguage = navigator.languages[0].split('-');
-		$langSelector.val(prefLanguage);
+    var languagesList = $langSelector.children('option').map(function() { return this.value; }).toArray();
+    prefLanguage = languagesList.indexOf(prefLanguage) !== -1 ? prefLanguage : fallbackLanguage;
+
+    $langSelector.val(prefLanguage);
```

Comment:
I am using `spaces` instead of `tabulations`, do we have a coding style to follow somewhere?

Reply:
I think the code is a mixed bag ATM.  Since there's a major update coming it's probably not worth spending a lot of time on it, but we ought to stick in an `.editorconfig` to nail down a common coding style.

Reply:
Agree with you thanks for your comment.

I raised an issue about https://github.com/decred/dcrweb/issues/132